Drop-In/src/main/java/com/braintreepayments/api/DropInActivity.java

```diff
@@ -122,7 +115,30 @@ public void handleOnBackPressed() {
 
         showBottomSheet();
     }
+    void fetchPaymentMethods() {
+        dropInClient.getVaultedPaymentMethods(this, (paymentMethodNonceList, error) -> {
+            if (error != null) {
+                onError(error);
+            } else {
+                dropInClient.getSupportedPaymentMethods(this, (paymentMethods, supportedPaymentMethodError) -> {
+                    if (supportedPaymentMethodError != null) {
+                        onError(supportedPaymentMethodError);
+                    }
+                    if (paymentMethods != null) {
+                        dropInViewModel.setSupportedPaymentMethods(paymentMethods);
+                    } else {
+                        onError(new BraintreeException("No payment methods supported"));
+                        return;
+                    }
+                    if (paymentMethodNonceList != null){
+                        dropInViewModel.setVaultedPaymentMethods(paymentMethodNonceList);
+                    }
+                    dropInViewModel.setHasFetchedPaymentMethods(true);
```

Comment:
We added `hasFetchedPaymentMethods` to `DropInViewModel` to allow updating the UI when both supported payment methods and vaulted payment methods have finished fetching.